pkg/retriever/directcandidatefinder.go

```diff
@@ -177,6 +177,12 @@ func (d *DirectCandidateFinder) retrievalCandidatesFromTransportsProtocol(ctx co
 					return
 				}
 			}
+		case "http":
```

Comment:
nice, so boost is reporting this as a protocol on probing? this means that boost is using libp2p's http implementation? or does this work somehow for a standard HTTP exposed port?